added What is Kafka primer to landing page and info to docs (#103)
tachgsfc/gcn.nasa.gov: app/routes/docs/faq.md

---
meta:
  title: GCN - FAQ
---

> # Frequently Asked Questions (FAQ)

Reviewer: ```suggestion
# Frequently Asked Questions
```

## What is Kafka?

[Apache Kafka](https://kafka.apache.org) is an open-source distributed event streaming platform used in both commerical web applications
and in recent years, it has been widely adopted by the astronomical community. Utilizing a rich development environment with
open-source and commericially available tools, GCN is running a cloud-based high-reliability Kafka broker that distributes

> GCN notices to users. Kafka is the platform of streaming data, and is agnostic of data format. However, GCN will transition to
> distributing native Kafka Avro records (see [roadmap](docs/roadmap)), which can be losslessly converted between text based JSON records and a binary format.

Reviewer: ```suggestion
GCN notices to users. Kafka transports the alerts, but it is agnostic about the data format.
```

Author: We reference JSON on the "one format, one protocol" image, so I was trying to provide context for that.
